src/components/APICatalog/APICatalog.tsx

```diff
@@ -38,7 +39,15 @@ const APICatalog: React.FC<APICatalogProps> = ({ apisToDisplay, limit }) => {
             key={apiDetails.id}
             xs={6}
           >
-            <ApplicationCard className={classes.card} onClick={handleOnCardClick(apiDetails)}>
+            <ApplicationCard
+              className={
+                clsx(
+                  classes.card,
+                  apiDetails.apiContract === t("fallbacks.noContract") && classes.contractlessAPIProduct,
```

Comment:
Do not compare with the translation if some changes this comparison in no longer valid. Instead check if the api contracts is empty.